satellite/satellitedb/overlaycache.go

```diff
@@ -195,6 +195,32 @@ func (cache *overlaycache) KnownUnreliableOrOffline(ctx context.Context, criteri
 	return badNodes, nil
 }
 
+// Reliable returns all reliable nodes.
+func (cache *overlaycache) Reliable(ctx context.Context, criteria *overlay.NodeCriteria) (nodes storj.NodeIDList, err error) {
+	// get reliable and online nodes
+	rows, err := cache.db.Query(cache.db.Rebind(`
+		SELECT id FROM nodes
```

Comment:
This concept of reliable came into existence essentially through code drift.  These ratios will go away and this check might become "NOT disqualified".  On the plus side, it means that this list will never have false positives.  On the negative side, this list might grow so large as to not be tenable in-memory.

Reply:
Yeah, I'm aware of the growth, but I think for now, this is better than hitting the DB hard.

Reply:
I totally second Egon's feeling